fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -702,6 +702,10 @@ private void forwardToMaster() throws Exception {
 
     @Override
     public void writeProfile(boolean isLastWriteProfile) {
+        if (context.getSessionVariable() != null
```

Comment:
better to set enable_profile = false in internal query context.

Reply:
done